src/hooks/useAccommodationsAndOffers.tsx/index.ts

```diff
@@ -46,6 +46,11 @@ export const useAccommodationsAndOffers = (props: SearchTypeProps | void) => {
     [data]
   );
 
+  const getAccommodationByHotelId = useCallback(
```

Comment:
maybe an idea to also rename the `getAccommodationById` so it's clear what the difference is.

I believe one is based on offerId and the other hotelId, right?